src/lib/m2/index.js

```diff
@@ -157,7 +157,11 @@ export default new r.Struct({
 
     this.transparencies.forEach((transparency) => {
       if (transparency.animated) {
-        animated = true;
+        if (transparency.keyframeCount > 1) {
+          animated = true;
+        } else if (transparency.firstKeyframe.value !== 32767) {
+          animated = true;
```

Comment:
I'm wondering whether the transparencies should be decoded as floats instead:

```
transparencies: new Nofs(new AnimationBlock(r.uint16le))
```

It seems to be a similar number as the Quat16Float conversion.